server/api/store.go

```diff
@@ -170,6 +170,37 @@ func (h *storeHandler) Delete(w http.ResponseWriter, r *http.Request) {
 	h.rd.JSON(w, http.StatusOK, nil)
 }
 
+func (h *storeHandler) SetState(w http.ResponseWriter, r *http.Request) {
```

Comment:
I am confused why we have these updates in this PR?

/cc @disksing

Reply:
I'm confused to. Seems you need to merge updates on master to your branch instead of pick them. @Connor1996